apps/vmq_diversity/priv/vmq_diversity.schema

```diff
@@ -70,6 +70,40 @@
   {default, 5},
   hidden]}.
 
+%% @doc Specify if the postgresql driver should use TLS or not.
+{mapping, "vmq_diversity.postgres.ssl", "vmq_diversity.db_config.postgres.ssl",
```

Comment:
should we call it tls?

Reply:
Calling it TLS is probably better, ssl is a replaced protocol. People still keep calling ssl to any encrypted TCP connection because they are used to the term

Reply:
I kept the SSL name as that's what we use everywhere else at the moment. So what do you think? Should we change this to TLS now and then the rest in VMQ 2.0 or shall we do all of them for VMQ 2.0 and keep SSL for now?

Reply:
Let's keep SSL